#goal definition
char[] objectPosition
---
#result definition
bool isOK
---
#feedback
int32[] sequence
